### PR TITLE
Synchronize output formatting with latest Mbed OS

### PR DESCRIPTION
**Pull request by marcuschangarm**
https://github.com/ARMmbed/mbed-os/pull/14576

### CODE REVIEW

**Review by teetak01 — APPROVED** [review]

File: mbed_lib.json [hunk 1]
```diff
@@ -14,7 +14,12 @@
         "fea-ipv6": {
             "help": "Used to globally disable ipv6 tracing features.",
             "value": null
+        },
+        "color-theme": {
+            "help": "Set color theme. 0 for readable, 1 for unobtrusive.",
+            "options": [0, 1],
+            "macro_name": "MBED_TRACE_COLOR_THEME",
```
Comment (mikter) [inline mbed_lib.json:21]: I think the default should be the one we had previously

Comment (marcuschangarm) [reply mbed_lib.json:21]: That would make it different from the default in Mbed OS. 

The main reason for this change, and to have the default be more visible, is to help our FAEs (and me) when our customers send us screenshots of their logs or do screen sharing during remote debug sessions, so we can actually read the debug output and help our customers.

**Comment by 0xc0170** [comment]
Can we proceed here, we would like to use this upstream repository with mbed-cli2.

**Comment by marcuschangarm** [comment]
@0xc0170 I think we'll have to wait for @teetak01 to get back from vacation, I don't have merge rights on this repo.

**Review by AnttiKauppila — APPROVED** [review]

**Comment by AnttiKauppila** [comment]
@0xc0170 Merged

**Pull request merged**